app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,21 @@
+module SessionsHelper
+  def login user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    id = session[:user_id]
+    current_user ||= User.find_by id: id if id
+    current_user
+  end
+
+  def logged_in?
+    current_user.present?
+  end
+
+  def log_out
+    session.delete :user_id
+    current_user = nil
+    current_user
```

Comment:
dòng này e viết có tácdụng gì

Reply:
ban đầu em không cho dòng cuối thì sẽ có lỗi Useless assignment to local variable (biến được gán nhưng không được sử dụng).
fixed ạ

Reply:
tìm cách bỏ dòng này đi e
tham khảo các bạn khác xem đoạn current_user này code ntn

Reply:
em đã edit log_out rồi mà nhỉ, em delete dòng 18